src/AspNet.Security.OAuth.Discord/AspNet.Security.OAuth.Discord.csproj

```diff
@@ -0,0 +1,38 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>ASP.NET Core security middleware enabling Discord authentication.</Description>
+    <VersionPrefix>1.0.0-beta2</VersionPrefix>
```

Comment:
Could you please update this file to match the other providers `.csproj`? We use a bunch of `.props` to reduce the number of elements in the individual projects, so it would be better to re-use them.